testv2/e2e/scenario_upgrade.go

```diff
@@ -25,6 +25,8 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const kubeoneVersionToInit = "1.4.3"
```

Comment:
Wouldn't it be better to get the latest release dynamically? I mean even here you'll have to update this to `1.4.4`.

Reply:
w.r.t https://github.com/kubermatic/kubeone/blob/master/install.sh#L24

Reply:
Hmm, that's something I'll be worried about in this case as well. For now, let's just stick to this hard coding. But would you please update this to `1.4.4`? ;)

Reply:
done

Reply:
Thanks